### PR TITLE
Fix/right menu

### PR DESCRIPTION
**Pull request by OmarDaSilva**
RightMenu Bar Update

**Problem:**
RightMenu bar needed an update due to the changes on the LeftMenu bar.

**Fix:**
- Consistent use of MenuTile component to match the left menu 
- Update to MenuTile & Tile component. Property size added so that icons can be centred properly and adds more consistency

Screenshot 2021-03-29 at 15 56 08

### CODE REVIEW

**Comment by OmarDaSilva** [comment]
This PR is ready for review

**Review by seanparsons — APPROVED** [review]

File: editor/src/components/menubar/menubar.tsx [hunk 1]
```diff
@@ -15,51 +14,49 @@ import { shareURLForProject } from '../../core/shared/utils'
...
 import { betterReactMemo } from '../../uuiui-deps'
-import { EditorAction, EditorDispatch } from '../editor/action-types'
+import { EditorAction } from '../editor/action-types'
 import { setLeftMenuTab, setPanelVisibility, togglePanel } from '../editor/actions/action-creators'
-import { EditorState } from '../editor/store/editor-state'
 import { useEditorState } from '../editor/store/store-hook'
 import { LeftMenuTab } from '../navigator/left-pane'
 
-const Tile = styled.div({
+interface TileProp {
```
Comment (Rheeseyb) [inline editor/src/components/menubar/menubar.tsx:30]: Minor, but can you please rename this to `TileProps`

File: editor/src/components/menubar/menubar.tsx [hunk 2]
```diff
@@ -15,51 +14,49 @@ import { shareURLForProject } from '../../core/shared/utils'
...
+const Tile = styled.div<TileProp>((props) => ({
   display: 'flex',
   flexDirection: 'column',
   justifyContent: 'center',
   alignItems: 'center',
-})
+  width: props.size,
+}))
 
 export interface MenuTileProps extends React.HTMLAttributes<HTMLDivElement> {
```
Comment (Rheeseyb) [inline editor/src/components/menubar/menubar.tsx:42]: Should this instead become `extends React.HTMLAttributes<HTMLDivElement>, TileProps`?

**Comment by OmarDaSilva** [comment]
The recent commit has addressed previous feedback and now ready for new feedback!

**Review by Rheeseyb — APPROVED** [review]

**Pull request merged**